src/test-helpers/createRtlTestApp.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { createRef } from 'react'
+import { act } from 'react-dom/test-utils'
+import { ReactWrapper } from 'enzyme'
+import { wrapInTestContext } from 'react-dnd-test-utils'
+
+import { initialize } from '../initialize'
+import { Provider } from 'react-redux'
+import { store } from '../store'
+import * as db from '../data-providers/dexie'
+import { clear } from '../action-creators'
+import { Await } from '../types'
+
+// components
+import AppComponent from '../components/AppComponent'
+import ErrorBoundaryContainer from '../components/ErrorBoundaryContainer'
+import TouchMonitor from '../components/TouchMonitor'
+import { render } from '@testing-library/react'
```

Comment:
Let's place the `render` import in the top section of imports. This section is just for components.